packages/nimble-components/src/wafer-map/specs/README.md

```diff
@@ -0,0 +1,199 @@
+As a first step, we are working on the "bloat removal" which is basically the removal of unwanted but exiting features (legend, tooltip etc.) and dependencies.
+
+Original component: [wafermap](https://dev.azure.com/ni/DevCentral/_git/op-web-portal?path=/libraries/wafermap&version=GBfeature/microServicesSplit)\
+Once the "bloat removal" is finished, the next step is to adapt the current 
+
+### Code submission strategy
+
+In order to make the process of submitting the re-written wafer map component easier for us and for the code reviewers as well, we propose the following bottom-up approach:
+* As an initial step, we submit a skeleton of the `nimble-wafer-map` component into the main branch. This will serve as a starting point on which the complete functionality will be built on.
+* Once the skeleton is in place, we start to work on and submit individual pieces of the functionality so it's not very overwhelming to review the code.
+
+### States
+
+We are not planning to allow the user to generate an “empty" wafer map, so we must ensure that data is provided when instantiating a new `nimble-wafer-map` component.
+The only action which we could consider as a "state transition" is the zoom functionality which was specified in the sections above.
+We expect that the end user be able to zoom in on the dies withing the wafer map and be able to zoom out by using the mouse wheel.
+Also, the user must be allowed to pan the zoomed in wafer map into any direction of its XY axis by holding the left mouse button and moving the mouse cursor around.
+
+Another interaction, which must be part of the component is the ability to identify the location of the mouse pointer on the wafer map die area.
+
+### Globalization
+
+Currently there will be no string data presented by this component. Might change in the future, but for now it will contain only numerical data.
+
+### Security
+
+N/A
+
+### Performance
+
+As mentioned in the "Risks" section, the biggest challenge from the performance standpoint is the representation of large number of datapoints and interacting with these datapoints in the browser.
+Currently the only thing we would like to achieve is to measure (benchmark) the time it takes a `nimble-wafer-map` component to be loaded with various number of datapoints.
+
+### Dependencies
+
+The only 3rd party dependency on which the current wafer map component depends on is the [D3(Data-Driven Documents)](https://d3js.org/) library.
+We use parts of this library in our zooming and rendering functionality.
+
+### Test Plan
+
+The current test plan is that moving forward with the development, we write unit tests which validate the core functionality.
+All tests written for this component should follow the standards of the nimble repository. 
+### Documentation
+We will write documentation on the `nimble-wafer-map` component, focusing on general usage, implementation and API. 
```

Comment:
It's good to clarify where the documentation will go.

Regarding implementation documentation, we've mostly documented the high level plan in the form of this spec but not gone into much more detail elsewhere (other than occasional inline comments). Do you think we'll need more docs for this component?

```suggestion
We will write Storybook documentation on the `nimble-wafer-map` component, focusing on general usage and API. 
```